test/unit/mdc-notched-outline/mdc-notched-outline.test.js

```diff
@@ -55,13 +55,14 @@ test('adapter#addClass adds a class to the root element', () => {
 test('adapter#removeClass removes a class to the root element', () => {
   const {root, component} = setupTest();
   component.getDefaultFoundation().adapter_.removeClass('foo');
-  assert.isFalse(root.classList.contains('foo'));
+  component.getDefaultFoundation().adapter_.removeNotchWidthProperty();
```

Comment:
Is the width property actually set on this element before these statements run? Otherwise we're not really verifying anything.